pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -444,6 +446,121 @@ def device_state(
         api_handler._response_check(res, "get machine status")
         return str(res.json()["state"])
 
+    @classmethod
+    def _get_calendar(
```

Comment:
This function is basically an API call, so I think we can move this to `QuantinuumAPI`

Reply:
Alec requested that make this similar to _available_devices, since we are using a slightly different URL

Reply:
@cqc-alec  FYI

Reply:
I have made the change now